cpp/core/operators/shuffle/splitter.h

```diff
@@ -25,13 +25,14 @@
 
 #include <random>
 
+#include "memory/ColumnarBatch.h"
+#include "operators/shuffle/SplitterBase.h"
 #include "operators/shuffle/type.h"
 #include "operators/shuffle/utils.h"
 #include "substrait/algebra.pb.h"
 
 namespace gluten {
-
-class Splitter {
+class Splitter : public SplitterBase {
```

Comment:
you can change it to`final : public` for the consideration of performance

Reply:
It is not final, fo example,  rangesplitter will extend it